### PR TITLE
New design correspondence

### PR DESCRIPTION
**Pull request by csKyou**

### CODE REVIEW

File: src/html/js/myboard.js [hunk 1]
```diff
@@ -42,24 +42,119 @@ getAppRequestInfo = function() {
     };
 };
 
+getAppRole = function(auth) {
+    if (auth == "read") {
+        // Currently we only allow role with read permission.
+        return 'CalendarViewer';
```
Comment (dixonsiu) [inline src/html/js/myboard.js]: Shouldn't it be MyBoardViewer?

File: src/html/locales/en/common.json [hunk 2]
```diff
@@ -14,6 +15,7 @@
     "sessionExpiredDialog": {
         "title": "$t(glossary:sessionExpiredDialog.title)",
         "message": "Session has expired.<br>$t(glossary:sessionExpiredDialog.action)",
+        "cancelBtn": "cencel",
```
Comment (dixonsiu) [inline src/html/locales/en/common.json]: "cancel" instead of "cencel".

File: src/html/templates/_list_template.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+<head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta http-equiv="Pragma" content="no-cache">
+    <meta http-equiv="Cache-Control" content="no-cache">
+
+    <!-- Bootstrap CSS -->
+    <link href="https://use.fontawesome.com/releases/v5.0.8/css/all.css" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/fullcalendar/3.9.0/fullcalendar.min.css">
```
Comment (dixonsiu) [inline src/html/templates/_list_template.html]: This template is for Personium Calendar not MyBoard.

File: src/html/templates/_shared_template.html [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+<head>
+   <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta http-equiv="Pragma" content="no-cache">
+    <meta http-equiv="Cache-Control" content="no-cache">
+
+    <!-- Bootstrap CSS -->
+    <link href="https://use.fontawesome.com/releases/v5.0.8/css/all.css" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/fullcalendar/3.9.0/fullcalendar.min.css">
```
Comment (dixonsiu) [inline src/html/templates/_shared_template.html]: This is for Personium Calendar not MyBoard.

**Review by dixonsiu — CHANGES_REQUESTED** [review]

File: src/html/locales/en/common.json [hunk 5]
```diff
@@ -58,6 +68,37 @@
...
+        "Member": {
+            "label": "Sharing Member",
+            "add": "Add Member"
+        },
+        "Send": {
+            "label": "Send sharing request",
+            "description": "Send sharing request with the following authority.",
+            "btnFooter": "Send"
+        },
+        "Response": {
```
Comment (dixonsiu) [inline src/html/locales/en/common.json]: Sharing.Received.label should it be "Received" instead of "Response"?

File: src/html/js/myboard.js (hunk 1 above)
Comment (csKyou) [reply src/html/js/myboard.js]: MyBoardViewer is correct

**Review by dixonsiu — CHANGES_REQUESTED** [review]

File: src/html/locales/en/common.json (hunk 5 above)
Comment (csKyou) [reply src/html/locales/en/common.json]: "Received" is correct

**Review by dixonsiu — APPROVED** [review]

**Pull request merged**